cls/Stage.py

```diff
@@ -66,27 +74,47 @@ def check_touched(self, pose_results, mp_pose, hand_results, frame, side):
 
                 RADIUS = 55  # radius around the palm
 
+                # 3D funcuality
                 if (side == 'RIGHT'):
-                    palm_center = calculate_center("RIGHT_PINKY", "RIGHT_INDEX", landmarks, mp_pose)
+                    right_pinky = landmarks_to_cv(landmarks[mp_pose.PoseLandmark.RIGHT_PINKY.value])
```

Comment:
extra code. you can do it like in this example: 
shoulder_Loc = landmarks_to_cv(landmarks[mp_pose.PoseLandmark.{side}_SHOULDER.value])